src/ngraph/runtime/reference/convert.hpp

```diff
@@ -41,6 +41,30 @@ namespace ngraph
                     out[i] = static_cast<char>(static_cast<bool>(arg[i]));
                 }
             }
+
+            template <typename TI>
+            void convert_float_to_bf16(void* arg, void* out, size_t count)
```

Comment:
Does `convert<TI, ngraph::bfloat16>` not work? If not, could we find a way to make it work?

FWIW, we do have casts between bfloat16 and float32.

Reply:
I think `static_cast` should be made to work in `bfloat16.hpp`

Reply:
ok, that seems to work...how about the other way round? do we support that as well?

Reply:
It should already work, though it _might_ only work if you're going directly to/from `float`. (I dunno when/if/how the whole promotion machinery kicks in for "custom" numeric types in C++...)

Reply:
I think there is an `operator float()` in `bfloat16`...